File: benchmarks/perl_php/php/strings.php
<?php $it=1000000; $r="";
for($i=0;$i<$it;$i++){ $s="Hello, World! ".$i; $s=preg_replace('/World/','Perl',$s,1); if($i==$it-1)$r=$s; }
echo "strings: $it, last=$r\n";

File: benchmarks/perl_php/strings.pl
my $it=1000000; my $r="";
for (my $i=0;$i<$it;$i++){ my $s="Hello, World! ".$i; $s=~s/World/Perl/; $r=$s if $i==$it-1; }
print "strings: $it, last=$r\n";
